slack/context.w

```diff
@@ -0,0 +1,93 @@
+bring "./message.w" as msg;
+bring "./util.w" as helpers;
+bring "./events.w" as events;
+
+/// The bahvioral interface of a sender
+pub inflight interface ISender {
+  inflight post(message: msg.Message): void;
+  inflight postText(message: str): void;
+}
+
+/// Represents the context of a slack channel
+pub inflight class Channel impl ISender {
+  /// The channel id
+  pub channel_id: str;
+  botToken: str;
+  
+  new (channel: str, botToken: str) {
```

Comment:
Originally that was my plan, but im not sure if Im dealing with a bug or limitation. Essentially within the postMessage() methods when trying to use `cloud.Secret` Ill receive an error that tells me I need to use `lift()` explicitly however, when I try to do an explicit lift I cant because the secret is an inflight object (since the class is an inflight class) 

Ill try and play with this a bit more to understand whats going on a bit better. Might call out for some help from our lifting experts.

Reply:
Had a chat with @Chriscbr looks like the core issue to passing the `cloud.Secret` around is related to: https://github.com/winglang/wing/issues/2730

I think for now its fine to proceed passing the token around as a string since its never exposed anywhere. Ill add a TODO: to revisit once we complete the mentioned issue (its part of stability milestone)